gapir/cc/interpreter.h

```diff
@@ -129,6 +129,9 @@ class Interpreter {
   // 7th to 13th MSB (both inclusive) of the opcode
   BaseType extractType(uint32_t opcode) const;
 
+  // Get 6 bit data out from an opcode located in the 26-20 LSB of the opcode.
+  uint32_t extract6bitData(uint32_t opcode) const;
```

Comment:
This function does not behave the same as the rest. It should have a different name.
Every other ExtractXBitData extracts from the LSB, this extracts from somewhere in the middle/

What do these 6 bits represent? Is is just arbitrary data? or is it something more./

Reply:
Sorry, this slipped through from another optimisation that is yet to be put up for code review.